### PR TITLE
fixes handcuffs not doing anything for clumsy people

### PR DESCRIPTION
**Pull request by Intigracy**
Needs #13211 first

You won't know if you're gonna get cuffed or they're gonna get cuffed until AFTER the cuffs are applied, so that you can't just walk away from them to avoid it.

:cl:
 * bugfix: Handcuffs now correctly apply to the person handcuffing when they're clumsy and pass the 50% chance.

### CODE REVIEW

**Review by PJB3005 — APPROVED** [review]

File: code/game/objects/items/weapons/handcuffs.dm [hunk 1]
```diff
@@ -69,6 +64,10 @@
 		else
 			feedback_add_details("handcuffs", "H")
 
+		if(clumsy_check(user) && prob(50))
+			to_chat(usr, "<span class='warning'>Uh... how do these things work?!</span>")
```
Comment (Exxion) [inline code/game/objects/items/weapons/handcuffs.dm]: `user`

**Review by Exxion — APPROVED** [review]

**Comment by Intigracy** [comment]
This will fail Travis fyi as the proc doesn't exist yet, keep your hands and feet away from the merge button.

**Comment by Intigracy** [comment]
fml

**Comment by PJB3005** [comment]
:clap:

**Comment by Exxion** [comment]
Dear fuck

**Comment by Intigracy** [comment]
You saw nothing.

**Pull request merged**